index.js

```diff
@@ -0,0 +1,183 @@
+var tableEl = document.getElementById('pic-table');
+var rowEl = document.createElement('tr');
+tableEl.appendChild(rowEl);
+var fieldEl = document.createElement('td');
+var picturesAll = [
+  bag,
+   banana,
+    bathroom,
+     boots,
+      breakfast,
+       bubblegum,
+        chair,
+         cthulhu,
+          dogduck,
+           dragon,
+            pen,
+             petsweep,
+              scissors,
+               shark,
+                sweep,
+                 tauntaun,
+                  unicorn,
+                   usb,
+                    watercan,
+                     wineglass
+                   ];
+
+var chooseRandomPic = function() {
+  var randomPic = picturesAll[Math.floor(Math.random() * picturesAll.length)];
+  // console.log(randomPic);
+  return randomPic;
+};
+
+chooseRandomPic();
+
+var threeRandomPics = function(){
+  console.log('!!!!!!!!!!!!!!!!!!!');
+  console.log('threeRandomPics,   oldRandomPics: ', oldRandomPics);
+  for (var i = 0; i < 3; i++) {
+    console.log(randomPics);
+    var randomPicture = chooseRandomPic();
+    console.log('randomPicture: ', randomPicture);
+    // if(i === 0) {
+    //   randomPics.push(randomPicture);
+      // console.log(randomPics);
+    // }else if(i !== 0) {
+    if(randomPics.includes(randomPicture) || oldRandomPics.includes(randomPicture)){
+      i--;
+        // console.log(randomPics);
+    }else{
+      randomPics.push(randomPicture);
+        // console.log(randomPics);
+    }
+  }
+};
+
+threeRandomPics();
+
+PictureSelect.prototype.createImage = function(i) {
+  var imageEl = document.createElement('img');
+  imageEl.setAttribute('src', this.imgSource);
+  imageEl.setAttribute('id', this.imgName);
+  imageEl.dataset.index = i;
+  this.views++;
+  return imageEl;
+};
+
+//-------------------------------clickcounter----------------------------
+
+//-------------------------------table----------------------------------
+
+function runCreateTable(){
+  for( var i = 0; i < randomPics.length; i++){
+    var fieldEl = document.createElement('td');
+    fieldEl.appendChild(randomPics[i].createImage(i));
+    rowEl.appendChild(fieldEl);
+  }
+};
+
+runCreateTable();
+
+function timesViewed(){
+
+}
+
+function incrementScores(index){
+  console.log('%&%*%(%)%%(%*&%&%*)');
+  console.log('incrementScores, randomPics: ', randomPics[index]);
+  randomPics[index].clicks++;
+}
+
+function resetBoard(){
+  rowEl.innerHTML = '';
+  oldRandomPics = randomPics;
+  randomPics = [];
+}
+
+function displayResults(){
+  var ulEl = document.getElementById('results');
+  for (var i = 0; i < picturesAll.length; i++) {
+    var listItem = document.createElement('li');
+    ulEl.appendChild(listItem);
+    listItem.textContent = 'You Clicked ' + picturesAll[i].imgName + ' ' + picturesAll[i].clicks;
+  }
+};
+
+function clickCount(event) {
+  console.log('event: ', event);
+  console.log(event.target.id);
+  console.log(fieldEl);
+  console.log(randomPics[event.target.dataset.index]);
```

Comment:
You should write yourself a comment so that you remember what is going on here, and what all these variables are.